examples/sessions/src/main.rs

```diff
@@ -81,25 +86,40 @@ where
 
         let headers = req.headers().expect("other extractor taken headers");
 
-        let cookie = if let Some(cookie) = headers
+        let session_cookie = if let Some(cookie) = headers
```

Comment:
I think the whole parsing logic here should just be replaced with `TypedHeader<Cookie>::from_request`.